lib/stamp/translator.rb

```diff
@@ -100,7 +99,7 @@ def two_digit_hour_emitter(token)
         Emitters::TwoDigit.new(:hour)
       else
         # 12-hour clock with leading zero
-        Emitters::TwoDigit.new(:hour, &HOUR_TO_12_HOUR)
+        Emitters::TwelveHour.leading_zero
```

Comment:
I'd rather just remove the constants in favor of factory methods. Maybe the constants just move to the different emitter classes and the factory methods just return the constants.